### PR TITLE
Add relative humidity and pressure input normalization adjustments to emulation

### PR DESCRIPTION
**Pull request by frodre**
This PR contains a few updates and experiments for Zhao-Carr emulation in effort to reduce biases in the Antarctic and tropics.

1. Added relative humidity transform to the ZC-specific list so it's available for direct input into the emulators
2. Added a `true_all` transform to the list of available `StdMethod` normalization strategies.  This computes the std deviation by centering using a mean calculated over **all** features instead of a per-feature centering done in `all`. This is used in tandem with `MeanMethod.all` to adjust the normalization for `air_pressure`  by preserving vertical information.  Provides a similar offline skill boost as directly inputting the RH.
3. Adds training/prognostic configurations for this round of experiments.

Other minor updates:
* Added `None` to the available list of normalization methods where data is neither scaled nor centered.
* Added a covenience method to filter the netcdf files going into a dataset via regexp matching, which sped up analysis of some antarctic diagnostics

Coverage reports (updated automatically):
- test_unit: [82%](https://output.circle-artifacts.com/output/job/a32ef4d9-7af0-4b69-91f7-4a5d6070d1be/artifacts/0/tmp/coverage/htmlcov-test_unit/index.html)

### CODE REVIEW

**Review by AnnaKwa — APPROVED** [review]
A couple suggestions and questions, nothing major to request.

File: workflows/diagnostics/tests/prognostic/emulation/_regtest_outputs/test_single_run.test_summary_function[antarctic_avg_column_integrated_cloud_5d_kg].out [hunk 1]
```diff
@@ -0,0 +1 @@
+[]
```
[the remark this answers is not in the document]
Comment (frodre) [reply workflows/diagnostics/tests/prognostic/emulation/_regtest_outputs/test_single_run.test_summary_function[antarctic_avg_column_integrated_cloud_5d_kg].out]: Good catch here.  The problem was missing data in the CDL definition at the specific times those summary metrics are looking for.

File: external/fv3fit/fv3fit/data/netcdf/load.py [hunk 2]
```diff
@@ -65,11 +68,15 @@ def nc_dir_to_tfdataset(
         shuffle: Randomly order the file ingestion into the dataset
         random_state: numpy random number generator for seeded shuffle
         cache: directory to cache datat at. The default is $pwd/.cache.
+        match: regexp match string to filter filenames
```
[the remark this answers is not in the document]
Comment (frodre) [reply external/fv3fit/fv3fit/data/netcdf/load.py]: Switched to use of `search` instead of match.  Thanks for pointing this out.

**Review by frodre — COMMENTED** [review]
Thanks for the review @AnnaKwa!

**Pull request merged**